blog.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>OUR BLOG</title>
+</head>
+
+<body>
+  <header>
+    <a href="index.html">
+      <img src="..." alt="Binabox Logo" />
+    </a>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="blog.html">Blog</a></li>
+        <li><a href="...">Catalog</a></li>
+        <li><a href="...">Collection</a></li>
+      </ul>
+      <input type="search" placeholder="Search..." />
+      <a href="...">Discord</a>
+      <a href="...">Connect</a>
+    </nav>
+  </header>
+
+  <main>
+    <section>
+      <!--Banner Title-->
+      <h2>Our blog</h2>
+      <div>
+        <ul>
+          <!--Group Buttons-->
+          <li><a href="index.html" target="_blank">Home</a></li>

```

Comment:
Use `target="_blank"` only for links to external websites. 
This opens the link in a new tab, which is useful for users, but using it for internal pages can be annoying and hurt user experience.

Also, add `rel="noopener noreferrer"` for security to prevent potential vulnerabilities through window.opener.